charts/descheduler/values.yaml

```diff
@@ -22,6 +22,7 @@ resources:
   #   cpu: 100m
   #   memory: 128Mi
 
+namespace: kube-system
```

Comment:
Curious why as Helm has its own `--namespace` flag.

```
  -n, --namespace string            namespace scope for this request
```

Reply:
Thanks for mentioning it. I changed the namespace values with `Release.Namespace` in the templates. This should make sense now for the `--namespace` flag.

>Release.Namespace: The namespace to be released into (if the manifest doesn’t override)
Ref: https://helm.sh/docs/chart_template_guide/builtin_objects/